src/pages/[platform]/build-a-backend/functions/examples/create-user-profile-record/index.mdx

```diff
@@ -92,16 +106,28 @@ const client = generateClient<Schema>({
 });
 
 export const handler: PostConfirmationTriggerHandler = async (event) => {
-  await client.models.UserProfile.create({
-    email: event.request.userAttributes.email,
-  });
-
+  await client.graphql(
+    {
+      query: `mutation createUserProfile($input: CreateUserProfileInput!) {
```

Comment:
we mention generating the graphql client code but it's not used here, is it needed?